### PR TITLE
Auto OCP-27495

### PR DESCRIPTION
**Pull request by xiaocwan**
@yapei Please review, thanks!

### CODE REVIEW

**Comment by xiaocwan** [comment]
Update and pass locally.

File: lib/rules/web/admin_console/4.4/common_ui_elements.xyaml [hunk 1]
```diff
@@ -133,6 +133,11 @@ check_sidebar_item:
   element:
     selector:
       xpath: //li[@class='co-resource-sidebar-item']//*[contains(text(),'<sidebar_item>')]
+close_modal:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/common_ui_elements.xyaml]: can be replaced with `click_close_icon`

File: lib/rules/web/admin_console/4.4/operator_hub.xyaml [hunk 2]
```diff
@@ -80,6 +80,59 @@ check_affected_pods_tab:
...
+    name: Marketplace
+    description: This Operator is purchased through Red Hat Marketplace
+  action: check_operator_description_on_overlay
+check_community_operator_description_on_overlay:
+  params:
+    name: Community
+    description: This is a community provided operator
+  action: check_operator_description_on_overlay
+check_operator_description_on_overlay:
+  action: click_checkbox_from_provider_type
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/operator_hub.xyaml]: all sub actions should be moved to `check_community_operator_description_on_overlay` and `check_marketplace_operator_description_on_overlay `

File: lib/rules/web/admin_console/4.4/operator_hub.xyaml [hunk 3]
```diff
@@ -80,6 +80,59 @@ check_affected_pods_tab:
...
+  - selector:
+      xpath:  //form[@data-test-group-name='providerType']
+    timeout: 30
+  - selector:
+      xpath: //*[contains(@class, 'pf-c-check__label') and text()='<text>']
+    op: click
+open_first_card_in_overlay:
+  elements:
+  - selector:
+      xpath: //a[contains(@class, 'pf-c-card')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/operator_hub.xyaml]: I think it should be `(//a[contains(@class, 'pf-c-card')])[1]`

**Pull request merged**